app/src/app/core/services/elasticsearch/data.service.ts

```diff
@@ -87,6 +87,27 @@ export class DataService {
     return this.performQuery<Artwork>(query);
   }
 
+    /**
+   * Fetches all child artworks of a specified iconclass
+   * Returns null if not found
+   * @param iconlass with '*' to fetch all artworks which start with the specified iconclass
+   * @param type if specified, it is assured that the returned entity has this entityType
+   */
+  public async findChildArtworksByIconography(iconclass: string, type?: EntityType): Promise<Artwork[]> {
+    const response = await this.http.get(this.searchEndPoint + '?q=iconographies:' + iconclass + '*').toPromise();
```

Comment:
> The REST-API (Elastic-Search) does not seem to be able to process iconographies like 11D121(+1). 11D121(+1)* 

Have you tried to escape brackets? Maybe ES interprets them as a regex group, therefore by `(+1)*` you would be searching for zero or more characters of + or 1 instead of a following wildcard. Furthermore `.+` could solve your issue concerning matches of the parent group. Maybe try them in a regex tester first.
In case you want to pre-filter those artworks with both, parent and child iconclass notation, you could use elasticsearch's `must_not` query, but unfortunately our own "patchworky" query builder does not support that. We should switch to BodyBuilder once and for all.

Reply:
I tested the rest call with %28 and %29 (uri encoded parentheses), but it doesn't seem to work so far.

Reply:
have you tried backslashes? Like 11D2\(+2\)*

Reply:
Already tried the other day. 
Mostly tried to get the rigth data sets from the cai-ddk rest endpoint so far. 
Currently I am not sure how parentheses are actually handled by elastic search.

- `https://cai-ddk-art-browser-staging.fbi.h-da.de/api/_search?q=iconographies:7995(+5)` - 7995 is just a random number, but for some reason it returns a **genre entity**?! Without (+5) no results are returned as intended.
- `https://cai-ddk-art-browser-staging.fbi.h-da.de/api/_search?q=iconographies:11D121(+5)` - Shouldn't return anything since it does not exist, but you are still getting the artwork entity 
- `https://cai-ddk-art-browser-staging.fbi.h-da.de/api/_search?q=iconographies:11D121(any_type_of_string)*` - with '*' and **any type of string within the parentheses** it is returning all results - basically the current error 

URI encoded parentheses yielded the same described results.

Reply:
This is really weird. I tried some things and a regex query seems to work. (At least, this one gives no random documents) 
`{
  "query": {
    "regexp": {
      "iconographies": {
        "value": "11D121\\(+5\\)",
        "flags": "ALL",
        "case_insensitive": true
      }
    }
  }
}`
Put this into the request body (Using the Querybuilder or whatever, I only tried it with insomnia). See [ES reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html)